Build/prepare_yocto_env.sh

```diff
@@ -0,0 +1,16 @@
+
+wget https://remote.mistywest.io/download/mh11/rzg2l/VerifiedLinuxPackage_v3.0.0/RTK0EF0045Z0021AZJ-v3.0.0-update2.zip
+wget https://remote.mistywest.io/download/mh11/rzg2l/VerifiedLinuxPackage_v3.0.0/RTK0EF0045Z13001ZJ-v1.2_EN.zip
+wget https://remote.mistywest.io/download/mh11/rzg2l/VerifiedLinuxPackage_v3.0.0/RTK0EF0045Z15001ZJ-v0.58_EN.zip
+wget https://remote.mistywest.io/download/mh11/rzg2l/VerifiedLinuxPackage_v3.0.0/oss_pkg_v3.0.0.7z
```

Comment:
Maybe we should not download this file by default but only when `buil.sh` is called with a certain argument. I also think the way it is used in `start.sh` should be reviewed, the idea was that it is being used if no download cache path is provided (before there was a build cache) but we have now added a build cache in the same parent directory. We should review this. Idea:

Provide an argument `-o` (?) to build the container in offline mode (with the .7z archive) and extract it when the container also is run in the offline mode `-o`, also. What do you think about this?

Reply:
Oh wow! the file is 2.2 GB. I only moved these lines from the dockerfile and I never checked its size. I also agree to use an argument for it.

Reply:
Keep in mind that the download is being cached in a docker layer. So it doesn't download it every time.